### DISCUSSION TITLE
Note links get reformatted in the chat responses and stop being internal links

### DISCUSSION THREAD

**Discussion by neneonline** [logancyang/obsidian-copilot#762] [Q&A] [answered]
I am trying to have copilot help me identify the right note links to add to my notes. The goal is, given a note, identify key concepts... look for those concepts among the notes in a certain folder... and add those links in the new version of the note.

I think it is doing a good job of identifying those related notes that I'd like to link inline... but when it produces the links to the notes... instead of doing it with the `[[note name]]` syntax... the links appear as `[note name](app://obsidian.md/note%20name)`.

I have tried to tweak my prompt to insist on using the double squared brackets syntax but I am starting to believe that it is the application layer the one that is reformatting the links that probably the LLM is producing the way they should.

**Comment by logancyang** [comment, marked as answer]
Hi @neneonline, thanks for the suggestion. This is an issue with markdown rendering in the Chat pane. The initial version of copilot had `[[note]]` directly in the AI response and it looks clickable but actually not, unless you copy the response back to the note. So I used the link instead. But as you said, the link becomes a nuisance when you copy it to note.

I'll come up with a better solution so we can keep `[[note]]` in the chat pane and make it clickable.

Created issue: https://github.com/logancyang/obsidian-copilot/issues/764

**Comment by neneonline** [comment]
Thanks for acknowledging it. 

I love your idea of copilot... you are on to something!! But a big part of that is bringing back copilot responses as a way to amplify notes... but if links are corrupted it no longer serves the purposes. I am not saying this to oppose to your response, because I think we are in agreement... just to encourage you to continue with the project!
